src/pages/PaniniCreator.tsx

```diff
@@ -31,52 +33,91 @@ export default function PaniniCreator(props: PaniniCreatorProps) {
+            Randomize Panini
+          </button>
         </div>
-      </Form>
-    </main>
+        <FormCard title="Configure Base">
+          <div className={styles.formSections}>
+            <SwipeSection removable={false} name="bread" title="bread" options={breadVariants}>
+              <img src="/src/images/wheat.svg" alt="wheatIcon" className={styles.wheatIcon}></img>
+            </SwipeSection>
+            <SelectSection
+              removable={true}
+              name="cheese"
+              title="cheese"
+              options={cheeseVariants}
+              maxElements={3}
+            ></SelectSection>
+            <SelectSection
+              removable={true}
+              name="meat"
+              title="meat"
+              options={meatVariants}
+              maxElements={2}
+            ></SelectSection>
+            <SwipeSection
+              removable={true}
+              name="dressing"
+              title="dressing"
+              options={dressingVariants}
+              maxElements={3}
+            ></SwipeSection>
+            <CheckboxButtonSection
+              name="vegetables"
+              title="vegetables"
+              options={vegetableVariant}
+            ></CheckboxButtonSection>
+          </div>
+        </FormCard>
+        <FormCard title="Configure Extras">
+          <div className={styles.formSections}>
+            <SelectSection
+              removable={true}
+              name="egg"
+              title="egg"
+              options={eggVariants}
+              maxElements={3}
+            ></SelectSection>
+            <CheckboxSection name="spread" title="spread" options={spreadVariant}></CheckboxSection>
+            <RadioSection name="serving" title="serving" options={servingVariant}></RadioSection>
+            <CheckboxSection name="topping" title="topping" options={toppingVariant}></CheckboxSection>
+          </div>
+        </FormCard>
+        <FormCard title="Finalize Order">
+          <div className={styles.formSections}>
+            <TextSection name={"name_panini"} title="name panini"></TextSection>
+            <CheckboxSection name="cutlery" title="cutlery" options={["Add to order"]}></CheckboxSection>
+            <CheckboxSection name="napkins" title="napkins" options={["Add to order"]}></CheckboxSection>
+          </div>
+          <div className={styles.formsSubmitInterfaceWrapper}>
+            <NavLink to={props.navTo} onClick={setOrderDataToTrue}>
+              <label className={styles.formsSubmitLabel}>
+                place order or start again
+                <input type="submit" className={styles.formsSubmit} value={"place order"} />
+              </label>
+            </NavLink>
+            <NavLink to="/panini_creator" onClick={resetOrderData} className={styles.formsResetNavLink}>
+              <button type="submit" className={styles.formsReset}>
```

Comment:
**nitpick:** isn't similar button created somewhere already? If yes, we should make it as a component